### ISSUE TITLE
Couldn't install additional dependencies

### ISSUE THREAD

**Issue by barcudcoch** [NexovaDev/UpdateHub#1342]
Getting the error code `1` when installing additional dependencies.

### Debug info:
App version: 2.4.0
WinGet version: 1.6.3482
Device ID: 7a115c33-edc9-407d-82db-2478fdb1cd3a
OS version: 19045.4046
Electron version: 29.0.1
Node version: 20.9.0
Chrome version: 122.0.6261.57

I run UpdateHub in my standard non-admin user. This causes problems with the operation of UpdateHub such as this error and updating Update Hub itself. Generally app installations that require admin rights are handled OK, providing the individual app update process  is well-written.
In summary, UpdateHub does not cope well with running as a non-admin user.

**Comment by barcudcoch** [comment]
I changed my usual login to be administrator but I had the same problem.
I Then completely uninstalled and reinstalled UPdateHub and receoved the same error. (I updated Winget too.)
(so not associated with beong a non-admin user.
